Mathlib/Dynamics/PeriodicPts.lean

```diff
@@ -628,22 +629,104 @@ namespace MulAction
 
 open Function
```

Comment:
I think you can remove a lot of `Function.` thanks to this.

Reply:
True, though I wonder if accidental name shadowing hasn't happened in the past

Reply:
I think lean will complain about ambiguous names.